Dispatchfile

```diff
@@ -10,10 +10,55 @@ load("github.com/mesosphere/dispatch-tasks/bump_charts/bump_charts@master", "bum
 git_clone_dirname = "kba-git-src"
 kind_cluster_name = "kba-kind-cluster"
 
+resource("artifacts", type="storage", params={
+    "type": "gcs",
+    "location": "s3://artifacts",
+    "dir": "yes"
+}, secrets={
+    "BOTO_CONFIG": k8s.corev1.SecretKeySelector(key="boto", localObjectReference=k8s.corev1.LocalObjectReference(name="s3-config"))
+})
+
 git_resource(git_clone_dirname, url="$(context.git.url)", revision="$(context.git.commit)")
+git_resource("ui-git", url="git@github.com:mesosphere/kommander.git", revision="v6.35.0")
 
 kind_cluster=make_kind_cluster(kind_cluster_name, git_clone_dirname)
 
+dindTask("dispatch-e2e-test",
+    inputs=[git_clone_dirname, "ui-git"],
+    outputs=["artifacts"],
+    steps=[
+        v1.Container(
+            name="fetch-master",
+            image="mesosphere/dispatch-dind:v0.5.2",
+            workingDir="/workspace/" + git_clone_dirname,
+            args=["git", "fetch", "origin", "master"]),
+
+        v1.Container(
+            name="run-e2e-tests",
+            image="mesosphere/kommander-ci:1045-0bc7580",
+            workingDir="/workspace/" + git_clone_dirname,
+            resources=k8s.corev1.ResourceRequirements(
+                limits={
+                    "cpu": k8s.resource_quantity("8000m"),
```

Comment:
😳

Reply:
Maybe that's a little too much, but I'd rather have it green first before optimizing.